lib/active_shipping/carriers/usps.rb

```diff
@@ -168,15 +168,15 @@ class USPS < Carrier
       /Delivery status information is not available/
     ]
 
-    ESCAPING_AND_SYMBOLS = /&amp;lt;\S*&amp;gt;/
-    LEADING_USPS = /^USPS/
+    ESCAPING_AND_SYMBOLS = /&lt;\S*&gt;/
+    LEADING_USPS = /^USPS /
     TRAILING_ASTERISKS = /\*+$/
     SERVICE_NAME_SUBSTITUTIONS = /#{ESCAPING_AND_SYMBOLS}|#{LEADING_USPS}|#{TRAILING_ASTERISKS}/
 
     def find_tracking_info(tracking_number, options = {})
       options = @options.update(options)
       tracking_request = build_tracking_request(tracking_number, options)
-      response = commit(:track, tracking_request, (options[:test] || false))
+      response = commit(:track, tracking_request, options[:test] || false)
```

Comment:
I don't like the fact that we have the `:test` option here in the first place. We should be able to pass `:test => true` (or preferably: `environment: :sandbox` or something) to the initializer, and not allow it to be overriden in every method.